### PR TITLE
i18n(zh-cn): update `v3.mdx` & `images.mdx` & `view-transitions.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

- update `v3.mdx`
- update `images.mdx` 
- update `view-transitions.mdx`

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/8646

### CODE REVIEW

File: src/content/docs/zh-cn/guides/upgrade-to/v3.mdx [hunk 1]
```diff
@@ -129,9 +129,9 @@ Astro v3.0 完全从代码库中删除了这个集成。Astro 图像的新解决
 
 移除 `@astrojs/image` 集成。你不仅需要卸载集成，还需要更新或删除任何导入语句和现有的 `<Image />` 和 `<Picture />` 组件。你可能还需要配置一个首选的默认图像处理服务。
 
-你可以在我们的 [图像指南](/zh-cn/guides/images/#移除-astrojsimage) 中找到完整的、逐步的删除旧图像集成的说明。
+你可以在我们的 [图像指南](#移除-astrojsimage) 中找到完整的、逐步的删除旧图像集成的说明。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/guides/upgrade-to/v3.mdx:132]:
```suggestion
你可以在我们的图像指南中找到 [完整的、逐步的删除旧图像集成的说明](#移除-astrojsimage)。
```

**Review by Nin3lee — CHANGES_REQUESTED** [review]
在 `v3.mdx` 中的第132行，勘误了一处链接的引用文本。

**Review by Nin3lee — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks! 🙌

**Pull request merged**